Type fixing fo typescript project. (#161)
jossmac/react-toast-notifications: README.md

## Toast Props

| Property                           | Description                                                                                           |
| ---------------------------------- | ----------------------------------------------------------------------------------------------------- |

> | appearance                         | Only used when no custom toast components are provided. One of `success`, `error`, `warning`, `info`. |

Reviewer: > Only used when no custom toast components are provided.

I'm not sure this is quite right, you could provide a custom toast that consumes this prop. Prefer something like:

```suggestion
| appearance | Used by the default toast. One of `success`, `error`, `warning`, `info`. |
```
